### PR TITLE
Set 🇨🇿  table/note font to liberation + remove \vspace hacks for 🇨🇿 🇪🇸 

### PR DESCRIPTION
**Pull request by tomaas-zeman**
Few things in this PR
- Using `liberation` font for Czech notes and tables to fix styling problems https://github.com/Heegu-sama/Homm3BG/issues/556#issuecomment-2161112358
- Czech and Spanish have different styles for subsections (https://github.com/Heegu-sama/Homm3BG/issues/556#issuecomment-2161221451) which resulted in many hacks in `.po` files. This removes all the hacks and sets a global setting for these two languages.
- To fix layouts after these changes, I had to make small text tweaks that need review from translators
  - 🇵🇱 ~~https://github.com/Heegu-sama/Homm3BG/pull/590/files#diff-6eed602a02961a0fb07e4600785dff184d06542ec1c8e1dc473f8f5e016168a1R147~~ ✅ 
  - 🇪🇸 https://github.com/Heegu-sama/Homm3BG/pull/590/files#diff-dad6a2ed870509cd54398fb23baed8becd09a01c17bda107b8d41f55709cc9f2R440
  - 🇪🇸 https://github.com/Heegu-sama/Homm3BG/pull/590/files#diff-dad6a2ed870509cd54398fb23baed8becd09a01c17bda107b8d41f55709cc9f2R445

pages

cs

### CODE REVIEW

File: translations/difficulty.tex/pl.po [hunk 1]
```diff
@@ -144,9 +144,9 @@ msgid ""
 "instead.} \\\\ \\end{tabularx} }"
 msgstr ""
 "\\hommtable{38}{ \\centering \\textbf{Tabela zasad opcjonalnych}\\\\ \\small "
-"{ \\color{white} Dzięki tym dodatkowym zasadom możesz urozmaicić swoje "
-"rozgrywki i modyfikować ich trudność.\\\\ } \\bigskip \\begin{tabularx}"
-"{\\linewidth}{>{\\hsize=0.3\\hsize\\linewidth=\\hsize}X "
+"{ \\color{white} Dodatkowymi zasadami możesz urozmaicić swoje rozgrywki i "
```
Comment (evilmint) [inline translations/difficulty.tex/pl.po:147]: Will `Dzięki dodatkowym zasadom` do? The suggested change feels less natural

**Comment by tomaas-zeman** [comment]
The build fails due to "too many passes". The issue is with the following text but I have no idea what's wrong

```
"The \\pagelink{Trading Post}{Mercadillo} te permite:\n"
"\\begin{itemize}\n"
"  \\item \\pagetarget{Trading}{comerciar}\\index{Comerciar} varios recursos con el juego de acuerdo a la \\hyperlink{Trade Table}{tabla de la contraportada},\n"
"  \\item \\textbf{eliminar una carta} de tu mano para venderla y ganar 1 \\includesvg[height=10px]{\\svgs/gold.svg},\n"
"    \\note{5}{Especialidades, Estadísticas, Habilidades y Flechas Mágicas iniciales \\textbf{no se eliminan} en el Mercadillo.}\\par\n"
"  \\item comprar una \\pagelink{War Machines}{Máquina de Guerra}, si tienes la expansión Murallas.\n"
"\\end{itemize}\n"
"En escenarios de \\textbf{Alianzas} y \\textbf{Cooperativos}, los jugadores pueden intercambiar Recursos y cartas siguiendo estas reglas:\n"
"\\begin{itemize}\n"
"  \\item En los Escenarios de Alianza, los aliados pueden intercambiar Recursos libremente en cualquier momento excepto durante el Combate.\n"
"  \\item En escenarios cooperativos, se pueden dar recursos a otros jugadores al visitar un puesto comercial.\n"
"  \\item En ambos tipos de escenario, los aliados pueden comerciar \\textbf{Hechizos} y \\textbf{Artefactos} de cualquier manera si tienen héroes en Campos adyacentes.\n"
"    nicamente \\textbf{las cartas en mano} se pueden intercambiar y debes dar y recibir la misma cantidad de cartas.\n"
"\\end{itemize}\n"
"\n"
```

File: translations/difficulty.tex/pl.po (hunk 1 above)
Comment (tomaas-zeman) [reply translations/difficulty.tex/pl.po:147]: That will do. It still fits. Thanks!

**Comment by qwrtln** [comment]
I think the existing font (on the left) looks better than what you're proposing (on the right):

image

Can't we have the left option everywhere, instead of forcing languages working correctly into the right option?

**Comment by qwrtln** [comment]
Or, at least make separate PRs for the following:

1. Czech fixes and typos
2. Spacing adjustments
3. Font fixes for tables and note boxes

**Comment by tomaas-zeman** [comment]
> I think the existing font (on the left) looks better than what you're proposing (on the right):
> 
> Can't we have the left option everywhere, instead of forcing languages working correctly into the right option?

The left option is `ptm` and doesn't work everywhere. The right one is the `default` font. Maybe we can just tweak the style a bit for notes so it looks more like `ptm` but I think we shouldn't keep `ptm` as it's causing issues for cs, ua, ru languages.

**Comment by tomaas-zeman** [comment]
> Or, at least make separate PRs for the following:
> 
> 1. Czech fixes and typos
> 2. Spacing adjustments
> 3. Font fixes for tables and note boxes

I thought about it but it's all connected - spacing and font changes both break the layouts and I don't want to fix it twice. There are actually no Czech typo fixes, it's part of shortening the text to fit the layout (as I did for `pl` and `es`)

**Comment by tomaas-zeman** [comment]
@qwrtln I changed the PR - it now only contains `subsection` changes for `es` and `cs` + `liberation` font for `cs` note boxes and tables.

File: sections/combat.tex [hunk 2]
```diff
@@ -209,6 +209,8 @@ \subsection*{Campaign and Cooperative Combat}
   \includegraphics[width=0.97\linewidth]{\art/dread_knight.jpg}
 \end{center}
 
+\clearpage
+
```
Comment (qwrtln) [inline sections/combat.tex:213]: Is it necessary?

Comment (tomaas-zeman) [reply sections/combat.tex:213]: It is. If there is a shorter text, the title from next page starts to creep in. It happened to me several times - this prevents that.

Screenshot 2024-06-12 at 18 52 26

**Review by qwrtln — APPROVED** [review]

**Comment by tomaas-zeman** [comment]
@Heegu-sama Could I ask for review here?

**Comment by tomaas-zeman** [comment]
@luischos Could you review changes in Spanish?

**Review by DarthGandalf — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Review by luischos — APPROVED** [review]

**Review by luischos — APPROVED** [review]

**Pull request merged**